### PR TITLE
Contest detail page

### PR DESCRIPTION
**Pull request by lintaoren**
Resolves judgels-dev/judgels#17

### CODE REVIEW

File: src/routes/labs/routes/contestDetail/ContestRegistrantTable/ContestRegistrantTable.scss [hunk 1]
```diff
@@ -0,0 +1,14 @@
+img {
```
Comment (fushar) [inline src/routes/labs/routes/contestDetail/ContestRegistrantTable/ContestRegistrantTable.scss]: Ini bakal ke-apply ke global `img`. Mungkin maksudnya dimasukin ke `.contest-registrant-table__avatar` ?

File: src/routes/labs/routes/contestDetail/ContestRegistrantTable/ContestRegistrantTable.scss [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+
+.contest-registrant-table__item {
+  margin-bottom: 10px;
+}
+
+.contest-registrant-table__avatar {
+  margin-right: 12px;
+}
+
+.contest-registrant-table__name {}
```
Comment (fushar) [inline src/routes/labs/routes/contestDetail/ContestRegistrantTable/ContestRegistrantTable.scss]: Hapus saja?

File: src/routes/labs/routes/contestDetail/ContestDetailPage/ContestDetailPage.tsx [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+      sideContent: DetailPageSide(),
+    };
+
+    return <DetailPage {...mockContestDetail} />;
+  }
+}
+
+export default withBreadcrumb('Contest Details Page')(ContestDetailPage);
+
+// export default function ContestDetailPage(props: ContestDetailPageProps) {
```
Comment (fushar) [inline src/routes/labs/routes/contestDetail/ContestDetailPage/ContestDetailPage.tsx]: Hapus commented code

File: src/routes/labs/routes/contestDetail/ContestRegistrantTable/ContestRegistrantTable.tsx [hunk 4]
```diff
@@ -0,0 +1,33 @@
+import * as React from 'react';
+import { Card } from '../../../../../components/Card/Card';
+import { Registrant } from '../../../../../modules/api/uriel/registrant';
+import Pagination from '../../../../../components/Pagination/Pagination';
+import './ContestRegistrantTable.css';
+
+export interface ContestRegistrantTableProps {
+  className?: string;
```
Comment (fushar) [inline src/routes/labs/routes/contestDetail/ContestRegistrantTable/ContestRegistrantTable.tsx]: Ini nggak dipakai

File: src/routes/labs/routes/contestDetail/ContestRegistrantTable/ContestRegistrantTable.scss (hunk 1 above)
Comment (lintaoren) [reply src/routes/labs/routes/contestDetail/ContestRegistrantTable/ContestRegistrantTable.scss]: oh betul jg. will change.

File: src/routes/labs/routes/contestDetail/ContestRegistrantTable/ContestRegistrantTable.tsx (hunk 4 above)
Comment (lintaoren) [reply src/routes/labs/routes/contestDetail/ContestRegistrantTable/ContestRegistrantTable.tsx]: ga perlu selalu nge pass class name ya mksdnya?

File: src/routes/labs/routes/contestDetail/ContestDetailPage/ContestDetailPage.tsx (hunk 3 above)
Comment (lintaoren) [reply src/routes/labs/routes/contestDetail/ContestDetailPage/ContestDetailPage.tsx]: noted.

**Pull request merged**